changelogs/fragments/7230-manual-cli-version-support.yaml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - onepassword lookup plugin - add CLI version lookup support to speed up auto-detection support (https://github.com/ansible-collections/community.general/pull/7230).
```

How about rewriting this as:
```suggestion
  - onepassword lookup plugin - allow to specify CLI major version as ``major_version`` to skip version auto-detection, which speeds up the plugin execution (https://github.com/ansible-collections/community.general/pull/7230).
```